src/main/java/mate/academy/springboot/datajpa/service/impl/CategoryServiceImpl.java

```diff
@@ -0,0 +1,32 @@
+package mate.academy.springboot.datajpa.service.impl;
+
+import java.util.NoSuchElementException;
+import mate.academy.springboot.datajpa.model.Category;
+import mate.academy.springboot.datajpa.repository.CategoryRepository;
+import mate.academy.springboot.datajpa.service.CategoryService;
+import org.springframework.stereotype.Service;
+
+@Service
+public class CategoryServiceImpl implements CategoryService {
+    private final CategoryRepository categoryRepository;
+
+    public CategoryServiceImpl(CategoryRepository categoryRepository) {
```

Comment:
you can use lombok for creating constructors. fix similar cases in other classes also

Reply:
I've already added lombok annotations but for some reason it is not shown in git